application/src/connect/JobSchedule.js

```diff
@@ -2,40 +2,35 @@ import _find from 'lodash/find'
 import _every from 'lodash/every'
 
 import { JOB_TYPES, JOB_STATUSES } from './consts'
-import { VERIFY_MODE, AGG_MODE, REWARD_MODE, TAX_MODE } from './const/Connect'
-
-const getJobTypeFromMode = mode => {
-  if (mode === VERIFY_MODE) return JOB_TYPES.VERIFICATION
-  if (mode === AGG_MODE) return JOB_TYPES.AGGREGATION
-  if (mode === REWARD_MODE) return JOB_TYPES.REWARD
-  if (mode === TAX_MODE) return JOB_TYPES.TAX
-
-  return JOB_TYPES.AGGREGATION
-}
 
 export const UNINITIALIZED = {
   isInitialized: false,
   jobs: [],
 }
 
-export const initialize = (member, recentJob, config) => {
+export const initialize = (member, updatedMember, config) => {
   let jobs = []
```

Comment:
This is just preference, but I think it would be easier to understand if we had jobs be a const, and instead of reassigning jobs we just pushed a value to the array. From my understand we never need to actually nuke what's in the array. We just need to add things to the end of it.

Reply:
I like the idea and I think it might work, my only concern is that if someone makes 2 connections in one widget session I don't know if those jobs might build up instead of getting cleared for each connection. Since this code needs to go live asap I think we should postpone this change and we can pair up and make the change and test if you'd like.